tests/ChainTest.php

```diff
@@ -89,8 +98,12 @@ public function providePrimitiveRulesData()
                     ],
                 ],
             ],
-            [
+            'break isArray rule on error' => [
                 new IsArray(),
+                new Each(function ($v) {
+                    /** @var Validator $v */
```

Comment:
Why not use typehint in signature?

Reply:
> "php": ">=5.4"

Reply:
Because of this: `"php": ">=5.4"`

Reply:
@vova07 Great. But

http://php.net/manual/en/functions.arguments.php#functions.arguments.type-declaration

Type hinting by class/interface name available since php 5.0